README.md

```diff
@@ -2,7 +2,7 @@
 
 **Status:**
 [![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux)](https://travis-ci.org/exonum/exonum)
-![CircleCI Build Status](https://img.shields.io/circleci/project/github/exonum/exonum.svg?label=MacOS)
+[![CircleCI Build Status](https://img.shields.io/circleci/project/github/exonum/exonum.svg?label=MacOS)](https://circleci.com/gh/exonum/exonum/tree/master)
```

Comment:
I'm a little bit late but
`https://img.shields.io/circleci/project/github/exonum/exonum.svg`
looks like we display the latest status of any build, not necessary of `master` branch.